### PR TITLE
feat: added asterisk for required field

### PR DESCRIPTION
**Pull request by djanov**
[CAR-8373](https://autoricardo.atlassian.net/browse/CAR-8373)

adding asterisk for the required field (floatingLabel)

### TODO

- [ ] add requreid for requreid fields in UL 
~~add requreid for requreid fields in DH~~

### CODE REVIEW

File: src/components/input/index.tsx [hunk 1]
```diff
@@ -59,6 +59,8 @@ interface PopupLabelProps extends InputProps {
 interface FloatingLabelProps extends InputProps {
   labelText?: string
   floatingLabel?: boolean
+  renderLabelPopup?: () => JSX.Element
```
Comment (Averethel) [inline src/components/input/index.tsx:62]: This looks unrelated, why is it needed?

Comment (djanov) [reply src/components/input/index.tsx:62]: I got a TS error, that I need to add it on line 102
image

Comment (djanov) [reply src/components/input/index.tsx:62]: it's probably because of the ternary nesting, and now we also have the required prop so that's why.

Comment (Averethel) [reply src/components/input/index.tsx:62]: It's because of the conditional, you're adding `renderLabalPopup` and `required` if there is `required` in other props. but the presence of `required` does not guarantee the presence of `renderLabelPopup`.

Comment (djanov) [reply src/components/input/index.tsx:62]: I just looked at the code, I don't know why I added here anything, I will just remove this part,

**Review by Averethel — APPROVED** [review]
Looks good to me. Can you make sure to add the new variant to the storybook before you merge?

**Comment by djanov** [comment]
> Looks good to me. Can you make sure to add the new variant to the storybook before you merge?

I added to the `standard ` in the storybook but sure will make a new one.
